assets/src/scss/blocks/TakeActionBoxout/TakeActionBoxoutEditorStyle.scss

```diff
@@ -27,7 +27,7 @@
   justify-content: center;
 
   img {
-    width: 100%;
+    width: 100% !important;
```

Comment:
I think that you can move the `boxout-image-container` class inside of the `boxout` class and avoid the `!important` statement.

Reply:
You are absolutely right! Updated 🙂